frontend/src/components/AdequacyCensusCharts/AdequacyCensusCharts.tsx

```diff
@@ -0,0 +1,133 @@
+import 'chart.piecelabel.js'
+import { isEmpty } from 'lodash'
+import CircularProgress from 'material-ui/CircularProgress'
+import * as React from 'react'
+// import { Doughnut } from 'react-chartjs-2'
+import { CENSUS_MAPPING } from '../../constants/census'
+import { ADEQUACY_COLORS } from '../../constants/colors'
+import { AdequacyMode, Method } from '../../constants/datatypes'
+import { withStore } from '../../services/store'
+import { summaryStatisticsByServiceAreaAndCensus } from '../../utils/data'
+import { formatNumber, formatPercentage } from '../../utils/formatters'
+import { getLegend } from '../MapLegend/MapLegend'
+import { StatsBox } from '../StatsBox/StatsBox'
+import './AdequacyCensusCharts.css'
+
+type Props = {
+  serviceAreas: string[],
+  censusCategory: string
+}
+
+/**
+ * Use circular legend patches instead of the default rectangles.
+ *
+ * TODO: Fix typings upstream in DefinitelyTyped/chart.js
+ */
+// (Chart as any).defaults.global.legend.labels.usePointStyle = true
+
+export let AdequacyCensusCharts = withStore('adequacies', 'method')<Props>(({ serviceAreas, censusCategory, store }) => {
+  if (isEmpty(store.get('adequacies'))) {
+    return <div className='AdequacyCensusCharts Flex -Center'>
```

Comment:
Done

Reply:
not done? I don't see a change here

Reply:
nvm I see you changed the name in the css file. You don't need to reference a custom css class here because you're not using anything custom. You're just doing Flex -Center, right? So just do className="Flex -Center".